url.bs

```diff
@@ -2483,6 +2483,8 @@ interface URL {
            attribute USVString search;
   [SameObject] readonly attribute URLSearchParams searchParams;
            attribute USVString hash;
+
+  USVString toJSON();
```

Comment:
Indeed, I haven't thought of that. How about `serializer<USVString>;`? It also has the advantage of being more language-independent, while there are precedents for `<>` meaning "produces" with `iterable`.